### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by kimrass14**
Created landing page: https://trello.com/c/Ht3qVBbE

Applied styling based on Miro: https://miro.com/app/board/o9J_lWMt_tk=/?moveToWidget=3074457360698655666&cot=14

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+// import { Link, Route } from 'react-router-dom'
```
Comment (jrapala) [inline src/components/App.js:1]: Can you please delete?

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,3 +1,36 @@
-const Home = () => <h1>Welcome to Voma</h1>;
+import styled from 'styled-components';
 
-export default Home;
+const StyledSection = styled.section`
+    width: 50%;
+    margin: 0 auto;
+    padding-top: 100px;
+    font-family: 'IBM Plex Sans', sans-serif;
```
Comment (jrapala) [inline src/pages/Home.jsx:7]: No need for this. We have a global style in place.

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+    font-size: 18px;
+`;
+
+const StyledImg = styled.img`
+    width: 50%;
+`;
+
+const StyledA = styled.a`
+    color: blue;
+    text-decoration: underline;
```
Comment (jrapala) [inline src/pages/Home.jsx:25]: Seems like this <a> styles are used everywhere. Can we add them to the Global Styles instead?

File: src/pages/Home.jsx [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+    font-family: 'IBM Plex Sans', sans-serif;
+`;
+
+const StyledH1 = styled.h1`
+    font-size: 48px;
+    font-weight: 600;
+`;
+
+const StyledParagraph = styled.p`
+    font-size: 18px;
```
Comment (jrapala) [inline src/pages/Home.jsx:16]: I think this should be 1rem instead

File: src/pages/Home.jsx [hunk 5]
```diff
@@ -1,3 +1,36 @@
...
-export default Home;
+const StyledSection = styled.section`
+    width: 50%;
+    margin: 0 auto;
+    padding-top: 100px;
+    font-family: 'IBM Plex Sans', sans-serif;
+`;
+
+const StyledH1 = styled.h1`
+    font-size: 48px;
```
Comment (jrapala) [inline src/pages/Home.jsx:11]: Can we use rem?

File: src/pages/Home.jsx [hunk 6]
```diff
@@ -1,3 +1,36 @@
...
+    color: blue;
+    text-decoration: underline;
+`;
+
+export default function Home() {
+    return (
+        <StyledSection>
+                <StyledH1>Sign in with Slack</StyledH1>
+                <StyledParagraph>We utilize Slack throughout our onboarding process. If you haven&#39;t already, please join our <StyledA href="https://code-for-chicago-slack-invite.herokuapp.com/">Slack</StyledA> and come back here to sign in.</StyledParagraph>
+                <StyledImg src="https://platform.slack-edge.com/img/sign_in_with_slack.png" srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x" alt="slack button" />
```
Comment (jrapala) [inline src/pages/Home.jsx:33]: Can you turn this into a button, please?
It just won't do anything yet though.

**Review by jrapala — CHANGES_REQUESTED** [review]
Left a few comments. Can you please also check if you have Prettier turned on? Most of these files are not formatted correctly.

File: src/pages/Home.jsx (hunk 2 above)
Comment (kimrass14) [reply src/pages/Home.jsx:7]: removed the `font-family`, but didn't see the other styling in global style

File: src/pages/Home.jsx (hunk 3 above)
Comment (kimrass14) [reply src/pages/Home.jsx]: added

File: src/pages/Home.jsx (hunk 4 above)
Comment (kimrass14) [reply src/pages/Home.jsx]: updated

File: src/pages/Home.jsx (hunk 5 above)
Comment (kimrass14) [reply src/pages/Home.jsx]: updated to 3rem

File: src/pages/Home.jsx (hunk 6 above)
Comment (kimrass14) [reply src/pages/Home.jsx]: button added

File: src/pages/Home.jsx (hunk 2 above)
Comment (jrapala) [reply src/pages/Home.jsx]: Here's a link to it: https://github.com/Code-For-Chicago/Voma-frontend/blob/75c072507e8e223bb83e8bb5fad9e801e0860d62/src/lib/GlobalStyles.js#L60

File: src/pages/Home.jsx [hunk 7]
```diff
@@ -1,3 +1,48 @@
...
+  font-size: 3rem;
+  font-weight: 600;
+`;
+
+const StyledParagraph = styled.p`
+  font-size: 1rem;
+`;
+
+const SlackButton = styled.button`
+  width: 30%;
```
Comment (jrapala) [inline src/pages/Home.jsx:19]: Can you please remove this width? It's making the button all blurry. It's fine if it doesn't match the design completely.

Screen Shot 2021-07-27 at 10 12 43 AM

File: src/pages/Home.jsx [hunk 8]
```diff
@@ -1,3 +1,48 @@
...
+      <StyledParagraph>
+        We utilize Slack throughout our onboarding process. If you haven&#39;t already, please join
+        our <a href="https://code-for-chicago-slack-invite.herokuapp.com/">Slack</a> and come back
+        here to sign in.
+      </StyledParagraph>
+      <SlackButton type="button">
+        <SlackImg
+          src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
+          srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"
+          alt="slack button"
```
Comment (jrapala) [inline src/pages/Home.jsx:43]: Can we improve this by changing this to `alt=""` and adding an `aria-label="Sign in with Slack"` to the `<button>` instead?

**Review by jrapala — CHANGES_REQUESTED** [review]
A few minor changes!

File: src/pages/Home.jsx [hunk 9]
```diff
@@ -1,3 +1,48 @@
...
+        We utilize Slack throughout our onboarding process. If you haven&#39;t already, please join
+        our <a href="https://code-for-chicago-slack-invite.herokuapp.com/">Slack</a> and come back
+        here to sign in.
+      </StyledParagraph>
+      <SlackButton type="button">
+        <SlackImg
+          src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
+          srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"
+          alt=""
+          aria-label="Sign in with Slack"
```
Comment (jrapala) [inline src/pages/Home.jsx:43]: This `aria-label` should go on the `<SlackButton>` please.

**Review by jrapala — CHANGES_REQUESTED** [review]

**Review by jrapala — APPROVED** [review]

**Pull request merged**